### PR TITLE
add ColorPaletter and ThemeProvider

### PR DESCRIPTION
**Pull request by IvanAleinikau**

### CODE REVIEW

File: lib/app/pages/fqa_page.dart [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+              );
+            },
+          ),
+        );
+      },
+    );
+  }
+
+  _scrollView(context, list) {
+    return RefreshIndicator(
```
Comment (Platton96) [inline lib/app/pages/fqa_page.dart:84]: add returned type

File: lib/app/pages/news_page.dart [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+              Icons.add,
+            ),
+            backgroundColor: ColorPalette.primaryColor,
+          ),
+        );
+      },
+    );
+  }
+  
+  _scrollView(context,list){
```
Comment (Platton96) [inline lib/app/pages/news_page.dart:96]: comment above

File: lib/app/pages/news_page.dart [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+                        list[index].url,
+                      ),
+                    ),
+                  ),
+                  Expanded(
+                    flex: 3,
+                    child: Padding(
+                      padding: const EdgeInsets.fromLTRB(5.0, 0.0, 0.0, 0.0),
+                      child: Column(
+                        crossAxisAlignment: CrossAxisAlignment.start,
```
Comment (Platton96) [inline lib/app/pages/news_page.dart:126]: too much method build separete

File: lib/app/pages/saved_news_page.dart [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+              },
+            ),
+          ),
+          drawer: const Menu(),
+        );
+      },
+    );
+  }
+
+  _scrollView(context, list) {
```
Comment (Platton96) [inline lib/app/pages/saved_news_page.dart:74]: also

File: lib/app/pages/settings_page.dart [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+              ],
+            ),
+          ),
+          drawer: const Menu(),
+        );
+      },
+    );
+  }
+
+  _switchLanguage(context){
```
Comment (Platton96) [inline lib/app/pages/settings_page.dart:70]: fix

File: lib/app/widgets/app_menu.dart [hunk 6]
```diff
@@ -143,22 +127,15 @@ class _MenuState extends State<Menu> {
...
-                  color: Colors.white,
-                  fontFamily: 'Oxygen',
-                  fontSize: 17,
-                ),
+                style: ThemeProvider.getTheme().textTheme.headline1,
               ),
               leading: const Icon(
                 Icons.question_answer,
-                color: Colors.white,
+                color: ColorPalette.whiteColor,
```
Comment (Platton96) [inline lib/app/widgets/app_menu.dart:138]: bad name

Comment (Platton96) [reply lib/app/widgets/app_menu.dart:138]: if you have dark theme

File: lib/app/pages/fqa_page.dart (hunk 1 above)
Comment (IvanAleinikau) [reply lib/app/pages/fqa_page.dart:84]: done

File: lib/app/pages/news_page.dart (hunk 2 above)
Comment (IvanAleinikau) [reply lib/app/pages/news_page.dart:96]: done

File: lib/app/pages/news_page.dart (hunk 3 above)
Comment (IvanAleinikau) [reply lib/app/pages/news_page.dart:126]: pulled out the card

File: lib/app/pages/saved_news_page.dart (hunk 4 above)
Comment (IvanAleinikau) [reply lib/app/pages/saved_news_page.dart:74]: done

File: lib/app/pages/settings_page.dart (hunk 5 above)
Comment (IvanAleinikau) [reply lib/app/pages/settings_page.dart:70]: fixed

File: lib/app/widgets/app_menu.dart (hunk 6 above)
Comment (IvanAleinikau) [reply lib/app/widgets/app_menu.dart:138]: changed name

**Pull request merged**